inference-engine/tests_deprecated/unit/engines/gna/gna_matcher.cpp

```diff
@@ -66,6 +66,33 @@ class NullAllocator : public IAllocator {
     }
 };
 
+template<class T>
```

Comment:
Can you move changed tests to the new test infra? We should migrate tests to the new infra?

Reply:
for unit tests we dont need your infrastructure - at all.